bundles/org.pitest.pitclipse.core/src/org/pitest/pitclipse/core/Mutators.java

```diff
@@ -49,48 +48,18 @@ public enum Mutators {
     NON_VOID_METHOD_CALLS("Non Void Method Call", "Removes method calls to non void methods. Their return value is replaced by the Java Default Value for that specific type"),
     PRIMITIVE_RETURNS("Primite Returns", "Replaces int, short, long, char, float and double return values with 0", true),
     REMOVE_CONDITIONALS("Remove Conditionals", "Removes all conditionals statements such that the guarded statements always execute"),
-    REMOVE_CONDITIONALS_EQ_IF("Remove Equal Conditionals If", "Remove equal conditions and replace with true, execute if part"),
-    REMOVE_CONDITIONALS_EQ_ELSE("Remove Equal Conditionals Else", "Remove equal conditions and replace with false, execute else part"),
-    REMOVE_CONDITIONALS_ORD_IF("Remove Order Checks If", "Remove order conditions and replace with true, execute if part"),
-    REMOVE_CONDITIONALS_ORD_ELSE("Remove Order Checks Else", "Remove order conditions and replace with false, execute else part"),
+    REMOVE_CONDITIONALS_EQUAL_IF("Remove Equal Conditionals If", "Remove equal conditions and replace with true, execute if part"),
+    REMOVE_CONDITIONALS_EQUAL_ELSE("Remove Equal Conditionals Else", "Remove equal conditions and replace with false, execute else part"),
+    REMOVE_CONDITIONALS_ORDER_IF("Remove Order Checks If", "Remove order conditions and replace with true, execute if part"),
+    REMOVE_CONDITIONALS_ORDER_ELSE("Remove Order Checks Else", "Remove order conditions and replace with false, execute else part"),
     REMOVE_INCREMENTS("Remove Increments", "Removes local variable increments"),
     EXPERIMENTAL_ARGUMENT_PROPAGATION("Experimentation Argument Propagation", "Replaces method call with one of its parameters of matching type"),
     EXPERIMENTAL_BIG_INTEGER("Experimental Big Integer", "Swaps big integer methods"),
+    EXPERIMENTAL_BIG_DECIMAL("Experimental Big Integer", "Swaps big decimal methods"),
```

Comment:
This should be "Decimal" not "Integer". Copy and paste? ;)

Reply:
Indeed, my bad!